scss/tests/mixins/_utilities-map.test.scss

```diff
@@ -0,0 +1,339 @@
+@mixin before {
+  $utilities: (
+    "flex": (
+      responsive: true,
+      property: flex,
+      values: (fill: 1 1 auto)
+    ),
+    "align": (
+      property: vertical-align,
+      class: align,
+      values:  baseline top
+    ),
+    "margin": (
+      property: margin,
+      class: margin,
+      values: auto
+    )
+  ) !global;
+}
+
+@import "../../mixins/utilities-map";
+
+@include describe("utilities") {
+
+  @include describe("utilities-get-options") {
+
+    @include it("Returns the options for the given utility") {
+      @include before ();
+      @include assert-equal(
+        utilities-get-options("flex"),
+        (
+          responsive: true,
+          property: flex,
+          values: (fill: 1 1 auto)
+        ));
```

Comment:
Wondering if we should use some hardcoded values or something like `map-get($utilities, "flex");`. Since it's done like we could keep it like that for now and change later if needed ?

Reply:
It would make the test just rewrite what the function does internally, which I was trying to avoid. It can be changed at a later stage if we feel like it though, like you said 😊